src/ledger/LedgerCloseMetaFrame.cpp

```diff
@@ -22,6 +22,10 @@ LedgerCloseMetaFrame::LedgerCloseMetaFrame(uint32_t protocolVersion)
         mVersion = 1;
     }
     mLedgerCloseMeta.v(mVersion);
+    if (mVersion == 1)
```

Comment:
Nit: I think it's cleaner to just check the close meta version directly using `mLedgerCloseMeta.v() == 1`.

Reply:
Done.